KEYCLOAK_SETUP.md

```diff
@@ -0,0 +1,56 @@
+# Setup Keycloak
+
+This document provides steps that need to be taken in order to successfully use Keycloak as IAM.
+
+Setup contains 9 steps:
+ - Adding a client.
```

Comment:
I would change this into an ordered list and use the appropriate numbers in each heading. Since "login as admin" is not a named step, I would integrate the related text into the introductory paragraphs.